src/components/Video/index.tsx

```diff
@@ -21,10 +24,45 @@ const Video: React.FC<VideoProps> = (props) => {
     env,
     autoplay = false,
     playthrough = false,
-    // default 16:9
-    aspectRatio = 0.5625,
+    viewportPercentage = 75,
+    shrinkToFit = false,
```

Comment:
aspect ratio was removed?

Reply:
0.5625 is the default

Reply:
the aspect ratio is passed in as a prop previously

Reply:
I'll add it back in as an override but undefined.  Better safe than sorry.  The only thing using this atm is video-promo-block.  Everything else uses VideoPlayer and I'm wondering if this shouldn't be converted to use that as well.

Reply:
`but undefined`? not sure I follow this

Reply:
Not going to default it to 0.5625, but leave it undefined and OR it with aspectRatio as defined by the calculation on DOM update